doc/ven_impl/ven_template/cookbook.md

```diff
@@ -0,0 +1,24 @@
+# CNTT - VI Cookbook
```

Comment:
If every Vendor has their own cookbook then need Vendor information (Vendor Name, Vendor Contact, Cookbook version, Cookbook date)

Reply:
@pgoyal01 great input. Each vendor has a folder and hence all vendor information are on README.md in the root of their folder (currently only the name)